logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/GetNotificationsUseCase.kt

```diff
@@ -103,7 +106,7 @@ internal class GetNotificationsUseCaseImpl internal constructor(
      * This `debounce` only for the case when we were Live and non-Live event comes helps to avoid such a scenario.
      */
     private fun Flow<IncrementalSyncStatus>.isLiveDebounced(): Flow<Boolean> =
-        this.map { it == IncrementalSyncStatus.Live }
+        this.map { it != IncrementalSyncStatus.FetchingPendingEvents }
```

Comment:
We can remove the `isLiveDebounced` completely, and just move the `map` to the `invoke` function.

Reply:
Ok